### PR TITLE
feat: add partial information from Pokémon Scarlet and Violet

### PR DESCRIPTION
**Pull request by bitomic**
- Adds all new abilities
- Adds all new abilities' names in English, Spanish, French, German and Italian
- Adds all new species; some data is missing
- Adds all new species's stats and EVs

### CODE REVIEW

**Comment by bitomic** [comment]
This PR won't/shouldn't close #771 because it lacks parts of the information.

**Review by Sethexus — APPROVED** [review]

File: data/v2/csv/pokemon_species.csv [hunk 1]
```diff
@@ -904,3 +904,106 @@ id,identifier,generation_id,evolves_from_species_id,evolution_chain_id,color_id,
 903,sneasler,8,215,109,4,6,,4,135,,0,,0,4,0,0,0,903,
 904,overqwil,8,211,106,1,3,,4,135,,0,,0,2,0,0,0,904,
 905,enamorus,8,,477,6,4,,8,3,,0,,0,1,1,1,0,905,
+906,sprigatito,9,,,Green,,,,45,50,0,20,0,,0,0,0,906,
```
Comment (Naramsim) [inline data/v2/csv/pokemon_species.csv]: Hi, the color can't be a string

File: data/v2/csv/pokemon_stats.csv [hunk 2]
```diff
@@ -5429,6 +5429,625 @@ pokemon_id,stat_id,base_stat,effort
 905,4,135,3
 905,5,80,0
 905,6,106,0
+
```
Comment (Naramsim) [inline data/v2/csv/pokemon_stats.csv:5432]: Can you remove this empty line?

File: data/v2/csv/pokemon_species.csv (hunk 1 above)
Comment (bitomic) [reply data/v2/csv/pokemon_species.csv]: I forgot to replace them with their ids - done!

File: data/v2/csv/pokemon_stats.csv (hunk 2 above)
Comment (bitomic) [reply data/v2/csv/pokemon_stats.csv:5432]: No problem, done!

**Review by Naramsim — APPROVED** [review]

**Comment by pokeapi-machine-user** [comment]
A [PokeAPI/api-data](https://github.com/PokeAPI/api-data) refresh has started. In 45 minutes the staging branch of [PokeAPI/api-data](https://github.com/PokeAPI/api-data/tree/staging) will be pushed with the new generated data. <br><br> The staging branch will be deployed in our [staging environment](https://staging.pokeapi.co/api/v2/) and you will be able to review the entire API. <br><br> A Pull Request ([master](https://github.com/PokeAPI/api-data/tree/master)<-[staging](https://github.com/PokeAPI/api-data/tree/staging)) will be also created at [PokeAPI/api-data](https://github.com/PokeAPI/api-data/pulls) and assigned to the PokeAPI Core team to be reviewed. If approved and merged new data will soon be available worldwide at [pokeapi.co](https://pokeapi.co/api/v2/).

**Pull request merged**